components/network/garage/Makefile

```diff
@@ -0,0 +1,57 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2025 Jiwon Na
+#
+
+BUILD_STYLE= justmake
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME= garage
+COMPONENT_VERSION= 2.0.0
+COMPONENT_SUMMARY= An open-source distributed object storage service tailored for self-hosting
+COMPONENT_PROJECT_URL= https://garagehq.deuxfleurs.fr
+COMPONENT_SRC= $(COMPONENT_NAME)
+COMPONENT_ARCHIVE= $(COMPONENT_SRC)-v$(COMPONENT_VERSION).tar.gz
+COMPONENT_ARCHIVE_URL= https://git.deuxfleurs.fr/Deuxfleurs/garage/archive/v$(COMPONENT_VERSION).tar.gz
+COMPONENT_ARCHIVE_HASH= sha256:10c50925c57ca9bd84e4b1395555f21fec7ab5acc74f1fa8ac785a30d62f80b6
+COMPONENT_FMRI= network/$(COMPONENT_NAME)
+COMPONENT_CLASSIFICATION= Web Services/Application and Web Servers
+
+include $(WS_MAKE_RULES)/common.mk
+
+CLONEY_MODE=copy
+
+PATH= $(PATH.gnu)
+
+COMPONENT_LICENSE= AGPL v3
+COMPONENT_LICENSE_FILE= LICENSE
+
+# https://www.illumos.org/issues/15767
+LD_Z_IGNORE=
+
+COMPONENT_BUILD_CMD = cargo build --release
+COMPONENT_BUILD_ENV += TERM=xterm
+COMPONENT_BUILD_ENV += SODIUM_USE_PKG_CONFIG=1
```

Comment:
If xterm is really needed for the build we need to add it as a build dependency.